### PR TITLE
Make Avatar size prop bindeable (Required by App Builder)

### PR DESCRIPTION
**Pull request by jcoitino**
Closes #  

### Additional information (check all that apply):
 - [ ] Bug fix
 - [x] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/avatar/avatar.component.ts [hunk 1]
```diff
@@ -256,6 +255,13 @@ export class IgxAvatarComponent implements OnInit, AfterViewInit {
         }
     }
 
+    /** @hidden @internal */
+    @HostBinding('class.igx-avatar--small') get _isSmallSize(): boolean { return this.size === 'small'; }
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/avatar/avatar.component.ts]:
```suggestion
    @HostBinding('class.igx-avatar--small')
    get _isSmallSize(): boolean {
        return this.size === IgxAvatarSize.SMALL;
    }
```

**Comment by kdinev** [comment]
@simeonoff Please review.

**Comment by kdinev** [comment]
@jcoitino There's a test that is failing after the changes. Could you please take a look?

**Comment by jcoitino** [comment]
Replaced by new one (PR #7014) adding bug and fixing test
NOTE: Tests was forced to pass hiding bug

**Pull request closed without merging**